temba/channels/views.py

```diff
@@ -1883,6 +1884,32 @@ def get_context_data(self, **kwargs):
             context['twitter_auth_url'] = auth['auth_url']
             return context
 
+    class ClaimFcm(OrgPermsMixin, SmartFormView):
+        class ClaimFcmForm(forms.Form):
+            title = forms.CharField(label=_('Notification title'))
+            key = forms.CharField(label=_('FCM Key'),
+                                  help_text=_("The KEY generated on Firebase Console when a new app is created."))
+
+        form_class = ClaimFcmForm
+        fields = ('title', 'key',)
+        title = _("Connect Firebase Cloud Messaging")
+        success_url = "id@channels.channel_configuration"
+
+        def form_valid(self, form):
+            org = self.request.user.get_org()
+
+            if not org:  # pragma: no cover
+                raise Exception(_("No org for this user, cannot claim"))
```

Comment:
These are spurious, there's no way we could get here without an org (because of permissions), we should remove all of these.